### ISSUE TITLE
Hello handshake with a Bonefish (another WAMP library) router/dealer failed

### ISSUE THREAD

**Issue by petten** [darrenjs/wampcc#1]
The code I used is similar to the example client.cc (and the sample code on your front page) here in git. The bonefish router was set up by someone else and proved working with a number of other clients in different languages (e.g., Ruby). I'm new to both the bonefish and wampcc. Anyhow, when I attempted to run the experimental client yesterday, I got two lines of logs from wampcc upon making a call to wampcc::wamp_session::initiate_hello():

20170414-20:16:17.708455 11484  WARN session #1 handhake error: server rejected handshake with error code 1 (serialised unsupported) (wamp_session.cc:586)
20170414-20:16:17.708800 11484  INFO session #1 closing (wamp_session.cc:1881)

The result was false of a subsequent call to wampcc::wamp_session::is_open().

**Comment by darrenjs** [comment]
Hi,

Thanks for the bug report.  I was able to reproduce.  The wampcc was not setting the frame buffer mask.  Additionally  bonefish does not support rawsocket/json combination (thus bonefish rejects the logon)

I have checked-in fixes into master, and put a new release 1.1.1.  The 'admin' program can now be used to connect to bonefish if the websocket protocol is selected.  For any of the examples to work, they also need to be modified to use `websocket` instead of `rawsocket`.

````shell
# start bonefish
bonefish  --realm "default_realm"  --websocket-port 9999 --debug 

# connect with wampcc admin program, invoke a procedure
admin  --proto=web -R default_realm  -c greeting  127.0.0.1 9999
````

**Comment by petten** [comment]
Fix confirmed. Thanks!
